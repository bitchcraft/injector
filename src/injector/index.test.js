/* eslint-env jest */
/* eslint-disable no-unused-vars */
import React from 'react';
import withInjector from './index';
import renderer from 'react-test-renderer';

const MyComponent = ({ children }) => React.createElement('span', { children });
MyComponent.displayName = 'MyComponent';
const stylesheet = values => ({});
const style = theme => ({});
const MyWrappedComponent = withInjector(stylesheet, style)(MyComponent);

test('inject()', () => {
	const component = renderer.create(
		<MyWrappedComponent>I am legion.</MyWrappedComponent>
	);
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
