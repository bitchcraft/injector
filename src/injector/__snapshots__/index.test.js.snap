// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inject() 1`] = `
<span>
  I am legion.
</span>
`;
